### PR TITLE
rename the hydra jobsets

### PR DESCRIPTION
**Pull request by cleverca22**
add a jobset to build cardano-sl master
add a dummy PR json to aid in testing

### CODE REVIEW

File: jobsets/default.nix [hunk 1]
```diff
@@ -1,4 +1,4 @@
-{ nixopsPrsJSON, cardanoPrsJSON, nixpkgs ? <nixpkgs>, declInput ? {} }:
+{ nixopsPrsJSON ? ./simple-pr-dummy.json, cardanoPrsJSON ? ./simple-pr-dummy.json, nixpkgs ? <nixpkgs>, declInput ? {} }:
```
Comment (domenkozar) [inline jobsets/default.nix]: Why are these defaults needed?

Comment (cleverca22) [inline jobsets/default.nix]: they make testing outside of hydra far simpler, you can just nix-build the file and get json you can diff

**Comment by domenkozar** [comment]
I think we also need to change hydra.conf configuration for status updates

**Comment by cleverca22** [comment]
https://github.com/input-output-hk/iohk-nixops/blob/master/modules/hydra-master.nix#L54

its configured to notify for any jobset that has an input called `jobsets`

**Comment by domenkozar** [comment]
@cleverca22 but doesn't `jobs = serokell:*` filter everything by the serokell project?

**Comment by cleverca22** [comment]
yeah, that will have to be fixed when we rename the project

**Pull request merged**